lightning/src/util/events.rs

```diff
@@ -150,6 +150,23 @@ pub enum Event {
 		/// The outputs which you should store as spendable by you.
 		outputs: Vec<SpendableOutputDescriptor>,
 	},
+	/// This event is generated when a payment has been successfully forwarded through us and a
+	/// forwarding fee earned.
+	PaymentForwarded {
+		/// The fee, in milli-satoshis, which was earned as a result of the payment.
+		///
+		/// Note that if we force-closed a channel while a forwarded HTLC was pending, the amount
+		/// the next hop claimed may have been rounded down, causing a higher fee than expected.
```

Comment:
s/a channel/a forwards channel ? A `next-hop channel`?

Reply:
Let me know what you think of the new wording